VenafiPS/Public/Get-TppIdentity.ps1

```diff
@@ -119,6 +134,12 @@ function Get-TppIdentity {
                         $response | Add-Member @{ 'Associated' = $associated.Identities }
                     }
 
+                    if (($response.IsGroup) -and ($GetMembers))  {
+                        $params.UriLeaf = 'Identity/GetMembers'
+                        $params.Body.Add("ResolveNested","1");
+                        $members = Invoke-VenafiRestMethod @params
+                        $response | Add-Member @{ 'Members' = $members.Identities}
```

Comment:
Member should have the same properties as the group, ID, IsGroup, Member, Name, Path.

Reply:
Pretty sure it does, I  am just returing whatever the GetMembers method returns and it does contain that data.